backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Tests/DelegationsControllerTest.cs

```diff
@@ -430,7 +442,11 @@ public async Task Post_DeletePolicies_Sucess()
             HttpResponseMessage response = await _client.PostAsync("authorization/api/v1/delegations/DeletePolicy", content);
 
             string responseContent = await response.Content.ReadAsStringAsync();
-            List<Rule> actual = (List<Rule>)JsonConvert.DeserializeObject(responseContent, typeof(List<Rule>));
+            var options = new JsonSerializerOptions
+            {
```

Comment:
If we actually need so specify this option for each test, then it move it to a common private property in the class, instead of having to set it up in each test